Description: avoid different versions of js-yaml
Origin: upstream, https://github.com/eslint/eslint/commit/cf5b6be
Bug: https://github.com/eslint/eslint/issues/15265
Author: Milos Djermanovic <milos.djermanovic@gmail.com>
Last-Update: 2022-01-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tests/bin/eslint.js
+++ b/tests/bin/eslint.js
@@ -354,10 +354,11 @@
             const child = runESLint(["--no-ignore", invalidConfig]);
             const exitCodeAssertion = assertExitCode(child, 2);
             const outputAssertion = getOutput(child).then(output => {
-                const expectedSubstring = ": bad indentation of a mapping entry at line";
-
                 assert.strictEqual(output.stdout, "");
-                assert.include(output.stderr, expectedSubstring);
+                assert.match(
+                    output.stderr,
+                    /: bad indentation of a mapping entry \(\d+:\d+\)/u // a part of the error message from `js-yaml` dependency
+                );
             });
 
             return Promise.all([exitCodeAssertion, outputAssertion]);
